client/src/handlebars/helpers.side-effects.js

```diff
@@ -390,6 +390,22 @@ function subject_name(subject){
 
 Handlebars.registerHelper('subj_name', subject_name);
 
+function subject_acronym_or_name(subject){
+
+  if (subject.is("dept")){
+    return (
+        !_.isEmpty(subject.abbr) ?
+        subject.abbr :
+        subject.fancy_name
+    );
```

Comment:
Huh, surprised the linter's not flagging this indentation. IMO, 397 should have two less spaces.

Reply:
:+1: